spec/index.bs

```diff
@@ -1016,13 +1016,35 @@ Applications must not rely on any other promise settlement ordering.
 
 ## Coordinate Systems ## {#coordinate-systems}
 
-- Y-axis is up in normalized device coordinate (NDC): point(-1.0, -1.0) in NDC is located at the bottom-left corner of NDC.
-    In addition, x and y in NDC should be between -1.0 and 1.0 inclusive, while z in NDC should be between 0.0 and 1.0 inclusive.
-    Vertices out of this range in NDC will not introduce any errors, but they will be clipped.
-- Y-axis is down in [=framebuffer=] coordinate, viewport coordinate and fragment/pixel coordinate:
-    origin(0, 0) is located at the top-left corner in these coordinate systems.
-- Window/present coordinate matches [=framebuffer=] coordinate.
-- UV of origin(0, 0) in texture coordinate represents the first texel (the lowest byte) in texture memory.
+Rendering operations use the following coordinate systems:
+
+* <dfn lt="NDC">Normalized device coordinates</dfn> (or NDC) have three dimensions, where:
+    * -1.0 &leq; x &leq; 1.0
+    * -1.0 &leq; y &leq; 1.0
+    * 0.0 &leq; z &leq; 1.0
+    * The bottom-left corner is at (-1.0, -1.0, z).
+    * Vertices outside these bounds will not introduce errors, but they will
+        be clipped.
+* <dfn noexport>Framebuffer coordinates</dfn> address the pixels in the [=framebuffer=]
+    * They have two dimensions.
+    * Each pixel extends 1 unit in x and y dimensions.
+    * The top-left corner is at (0.0, 0.0).
+    * x increases to the right.
+    * y increases down.
+    * See [[#render-passes]] and [[#rasterization]].
+* <dfn>Viewport coordinates</dfn> combine [=framebuffer coordinates=] in x and y dimensions,
+    with depth in z.
+    * Normally 0.0 &leq; z &leq; 1.0, but this can be modified by setting {{RenderState/[[viewport]]}}.`minDepth` and `maxDepth` via
```

Comment:
Note: it took me while to convince myself that there is not predefined orientation of z.

It all depends on how you configure your depth clear value and depth compare op.
I didn't say anything about that here; it kind of seems the wrong place to do it?

Reply:
Indeed there is not. That is what there was a bunch of haranguing about in the F2F about about defaults for the depth clear value and depth compare op. There's still a possibility we'll add defaults in the future but right now there isn't even a default z orientation.

Reply:
Agreed with not mentioning it.